### ISSUE TITLE
Page navigation not respecting location

### ISSUE THREAD

**Issue by jeanconn** [sot/acq_stat_reports#7]
It look like the UP_TO_MAIN, PREV, NEXT links on the text output for #6 aren't self-contained to go to test output.   I think that used to work, so might be something to fixup in the "wrap-up" PR planned after plots.

https://icxc.cfa.harvard.edu/aspect/test_review_outputs/acq_stat_reports/pr-6/web/2024/YEAR/index.html

**Comment by javierggt** [comment]
I don't think there is a way to do this automatically, and is something I did not change. Even before I started making changes, the way this worked was through a command-line argument `--url` which defaults to `/mta/ASPECT/acq_stat_reports/`.

This path is the absolute path of the web page, which is not know from the filesystem. What looks as `/mta/ASPECT/acq_stat_reports/` in the browser is actually `SKA/www/ASPECT/acq_stat_reports` on the file system.

The output of PR #6 is in icxc, so the paths are different. What looks as `/aspect/test_review_outputs/acq_stat_reports/pr-6` on the browser is actually `SKA/www/ASPECT_ICXC/test_review_outputs/acq_stat_reports/pr-6`.

One could add code to check some standard paths and do replacements, but there are many possibilities:
- SKA could be in proj/sot,
- it could be some environment's SKA,
- it could be some link
- or it could have been passed via the command line, in which case all bets are off.

The bottom line is that it is harder to make these changes than to have a command-line argument with a good default.

I fixed the output of #6 so the URL is correct (see the description in #6)

**Comment by javierggt** [comment]
Not an issue anymore
